torch_ipex/csrc/cpu/dbl/Common.h

```diff
@@ -12,6 +12,31 @@ namespace comm {
 dil::tensor dil_tensor_from_dense(const at::Tensor& tensor);
 at::Tensor dil_tensor_to_dense(const at::Tensor& tensor);
 void reorder_to_bf16_for_mix_prec(const at::Tensor& tensor);
+
+/**
+ * Reorder the input tensor to the specified scalar type.
+ *
+ * @param[in] tensor          The tensor to be reordered to the spcified scalar type
+ * @param[in] dst_scalar_type The scalar type which the shade buffer of the ipex tensor will be reordered to. It should
+ *                            be at::kBFloat16 or at::kFloat
+ */
+void reorder_to_dtype(const at::Tensor& tensor, at::ScalarType dtype);
+
+/**
+ * Reorder the input tensor to the expected descriptor.
+ *
+ * @param[in] tensor        The tensor to be reordered to the spcified scalar type
```

Comment:
nit: `The tensor to be reordered to the spcified scalar type` -> `The tensor to be reordered to the extected dil tensor descriptor`